app/src/main/res/layout/item_history.xml

```diff
@@ -1,61 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<androidx.cardview.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     app:cardCornerRadius="10dp"
     app:cardElevation="9dp"
-    android:layout_margin="10dp">
+    android:layout_marginHorizontal="8dp"
+    android:layout_marginTop="8dp">
```

Comment:
change margin to app:cardUseCompatPadding="true" for the padding on card view into recylerView